cmd/clusterctl/client/config/reader_viper.go

```diff
@@ -61,11 +61,11 @@ func (v *viperReader) Init(path string) error {
 	viper.AllowEmptyEnv(true)
 	viper.AutomaticEnv()
 
-	// If a path file is found, read it in.
-	if err := viper.ReadInConfig(); err == nil {
-		log.V(5).Info("Reading configuration", "File", viper.ConfigFileUsed())
+	err := viper.ReadInConfig()
+	if err != nil {
+		return err
 	}
-
+	log.V(5).Info("Reading configuration", "File", viper.ConfigFileUsed())
 	return nil
```

Comment:
I did it this way instead of just 
```
log.V(5).Info("Reading configuration", "File", viper.ConfigFileUsed())
return viper.ReadInConfig()
``` 
because viper doesn't set the `configFile` variable until `viper.ReadInConfig()` is called. That is, in the above scenario, it would print `Reading configuration File=""`